auspice/server/fetch.js

```diff
@@ -0,0 +1,22 @@
+const __fetch = require("make-fetch-happen");
+const utils = require("./utils");
+
+const _fetch = __fetch.defaults({
+  cache: "default",
+  cacheManager: process.env.FETCH_CACHE || "/tmp/fetch-cache",
```

Comment:
Is the plan to configure this variable for the heroku servers?

Reply:
No, the default should be fine. I did intend to document it but forgot, so I just pushed a commit noting it in the infrastructure docs.